test/browser_tests/spec_suites/careers-application-process.js

```diff
@@ -52,4 +52,31 @@ describe( 'The Application Process Page', function() {
     expect( page.relatedLinksSection.isPresent() ).toBe( true );
   } );
 
+  describe( '(Video Player)', function() {
+    it( 'should be present', function() {
+      expect( page.videoPlayer.isPresent() ).toBe( true );
+    } );
+
+    it( 'display show an image when the page loads', function() {
+      expect( page.videoPlayerImageContainer.isDisplayed() ).toBe( true );
+      expect( page.videoPlayerVideoContainer.isDisplayed() ).toBe( false );
+    } );
+
+    it( 'should show the video when the play button is clicked', function() {
+      page.videoPlayerPlayButton.click();
+      expect( page.videoPlayerVideoContainer.isDisplayed() ).toBe( true );
+      expect( page.videoPlayerImageContainer.isDisplayed() ).toBe( false );
+    } );
+
+    it( 'should attach an iframe when the play button is clicked', function() {
+      expect( page.getVideoPlayerIframe().isPresent() ).toBe( true );
```

Comment:
Fixed in https://github.com/cfpb/cfgov-refresh/pull/2103/commits/8af027e56f54ecf29aec8648bac4205ed0570b9b.